packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -20,7 +20,9 @@ type Props = {|
     /**
      * The items in this select.
      */
-    children?: Array<React.Element<OptionItem>>,
+    children?: Array<
+        React.Element<OptionItem> | false | typeof undefined | null,
```

Comment:
`Array<?(React.Element<OptionItem> | false)>`?

Reply:
Shouldn't there be a `typeof` in there, e.g.
```
Array<?React.Element<typeof OptionItem> | false>
```

Reply:
My bad, I did realize that `OptionItem` was using a `typeof` import at the top of the file.